### PR TITLE
FEATURE: Extract transition timings to constants

### PR DESCRIPTION
**Pull request by markusguenther**
Extracting the transition timings to constants.

Resolves: #801

### CODE REVIEW

**Comment by markusguenther** [comment]
Trigger styleci again

**Comment by dimaip** [comment]
Could we maybe put `transitionVars` into the `styleVars`?

**Comment by markusguenther** [comment]
Yes we can ;)

File: packages/build-essentials/src/config/transitionConfig.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+const upperFirst = require('lodash.upperfirst');
+
+const config = {
+    transition: {
+        fast: '.1s',
+        default: '.2s',
+        medium: '.3s',
```
Comment (dimaip) [inline packages/build-essentials/src/config/transitionConfig.js]: Btw, these kind of refactoring really raise the UX-related questions, e.g. is `.2s` really distinguishable from `.3s`?
But this should obviously be taken care of by the UX guild and not here.

**Comment by markusguenther** [comment]
@dimaip You was to fast... the first was just resolving conflicts with the master via GitHub. But that did not affect the transition constants ;)

**Comment by dimaip** [comment]
Ouch sorry, overlooked it :blush:

File: packages/build-essentials/src/config/transitionConfig.js (hunk 1 above)
Comment (markusguenther) [reply packages/build-essentials/src/config/transitionConfig.js]: To be honest I see no difference ... in my point of view we could maybe drop 200ms and use 300ms as default... then we have just slow, fast and default.

Comment (markusguenther) [reply packages/build-essentials/src/config/transitionConfig.js]: Or 250 as default and 100 as fast and 500 as slow.

Comment (dimaip) [reply packages/build-essentials/src/config/transitionConfig.js]: Agreed.

Comment (markusguenther) [reply packages/build-essentials/src/config/transitionConfig.js]: Adjusted that now.

**Pull request merged**